git-rpc-server/git_rpc_server/server.py

```diff
@@ -0,0 +1,27 @@
+from jsonrpc import JSONRPCResponseManager, dispatcher
+from jsonrpc.exceptions import JSONRPCError
+import os
+from werkzeug.wrappers import Request, Response
+
+from git_rpc_server import config
+from git_rpc_server.methods import git, resources
+
+
+@Request.application
+def application(request):
+    """Listen for incoming requests on /jsonrpc"""
```

Comment:
Is this actually still true? If so, who responds to the probes which are hitting "/"?

Reply:
It responds to all paths - I think this makes sense as the json rpc server should always respond with 200. This follows the example from the `json-rpc` library so I will leave it as is and update the doc string only.